homeassistant/components/services.yaml

```diff
@@ -227,6 +227,10 @@ logger:
   set_level:
     description: Set log level for components.
 
+system_log:
```

Comment:
Let's not add more services to this file. Instead, move `system_log.py` to `system_log/__init__.py` and add `system_log/services.yaml`. Loading a 500 line YAML is slow.

(maybe we just need 1 services.yaml file for EVERYTHING that is loaded and removed from memory after startup of HASS is done but that's a story for another day…)

Reply:
Sounds good, I'll fix that! 👍